config.yaml

```diff
@@ -2,27 +2,27 @@
 
 # If submitting to a different project to your default, uncomment line below 
 # and change project code as appropriate; also set shortpath below
-project: x77
+project: tm70
```

Comment:
I guess don't change this until https://github.com/COSIMA/access-om3/issues/114 is implemented

Reply:
Reverted